src/pages/Thanks/ThanksPage.vue

```diff
@@ -302,6 +303,14 @@ export default {
 		userHasLentBefore(hasLentBefore);
 		userHasDepositBefore(hasDepositBefore);
 
+		// MARS-246 Hotjar user attributes
+		setHotJarUserAttributes({
+			userId: this.userId,
+			hasEverLoggedIn: true,
```

Comment:
Can this page only be accessed by logged in user?

Reply:
`this.userId` isn't defined on the thanks page. You can conditionally check for it though...
`userId: !data?.my?.userAccount?.id ?? null`